packages/terra-toggle/src/Toggle.jsx

```diff
@@ -77,7 +79,7 @@ class Toggle extends React.Component {
 
     if (isAnimated) {
       body = (
-        <div className="terra-Toggle-content" ref={(div) => { this.contentContainer = div; }}>
+        <div className={styles['toggle-content']} ref={(div) => { this.contentContainer = div; }}>
```

Comment:
I agree, lets use `cx` everywhere instead of accessing style block. Its consistent and cleaner IMO